extensions/dicom-video/src/viewports/OHIFCornerstoneVideoViewport.tsx

```diff
@@ -30,6 +32,9 @@ function OHIFCornerstoneVideoViewport({ displaySets }) {
         controlsList="nodownload"
         preload="auto"
         className="w-full h-full"
+        crossOrigin={(appConfig.videoViewport && appConfig.videoViewport.crossOrigin) ? appConfig.videoViewport.crossOrigin : undefined}
+        autoPlay={(appConfig.videoViewport && appConfig.videoViewport.autoPlay) ? appConfig.videoViewport.autoPlay : false}
+        muted={(appConfig.videoViewport && appConfig.videoViewport.autoPlay) ? appConfig.videoViewport.autoPlay : undefined}
```

Comment:
I don't like how we are adding viewport-specific config now to the config in the config. 
I'm pretty sure the new customizationService by @wayfarer3130 can handle these sort of stuff. No?

Reply:
oK, I looked at the docs and the service and the intent of the service might be solving for this. However, I am not clear how to configure the config values in the service still. I see there is supposed to be support for it but I don't see the service used anywhere. 
I am thinking I can create a "getCustomizationModule" in the dicom-video extention and set these as default values there so others can extend them. if that's the case it makes sense to me. 

can someone show me what the format of the value looks like to get this setup?

Reply:
@md-prog can we get some help/insight from you here

Reply:
const videoViewportConfig = customizationService.getModeCustomization("videoViewportConfig", {
  autoPlay: false, muted: false })

then just use videoViewportConfig.  You can define a custom version of it in the APP_CONFIG file.